Modules/Core/Mesh/test/itkSimplexMeshAdaptTopologyFilterTest.cxx

```diff
@@ -19,10 +19,17 @@
 #include "itkTriangleMeshToSimplexMeshFilter.h"
 #include "itkSimplexMeshAdaptTopologyFilter.h"
 #include "itkDefaultDynamicMeshTraits.h"
+#include "itkTestingMacros.h"
 
 int
-itkSimplexMeshAdaptTopologyFilterTest(int, char *[])
+itkSimplexMeshAdaptTopologyFilterTest(int argc, char * argv[])
 {
+  if (argc != 3)
+  {
+    std::cerr << "Missing parameters." << std::endl;
+    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv) << " threshold selectionMethod" << std::endl;
```

Comment:
Not a request, just a side note, FYI.

In _itkNameOfTestExecutableMacro_

```cpp
#define itkNameOfTestExecutableMacro(argv)                                                                             \
  [argv](const std::string & functionName) {                                                                           \
    return ((argv == nullptr) || (argv[0] == nullptr) || (argv[0][0] == '\0')) ? ("<" + functionName + " executable>") \
                                                                               : argv[0];                              \
  }(__func__)
```

the expression `((argv == nullptr) || (argv[0] == nullptr) || (argv[0][0] == '\0'))` will never evaluate to _true_. If _argv_ is there, _argv[0]_ is always name of the executable, isn't it?
Also for usage with intentional _nullptr_ (to print name of function) it is problematic, with GCC 8.3
`std::cout << tkNameOfTestExecutableMacro(nullptr)<< std::endl;`
or
`std::cout << itkNameOfTestExecutableMacro((char**)nullptr) << std::endl;`
doesn't compile, only
```cpp
	char ** x = nullptr;
	std::cout << itkNameOfTestExecutableMacro(x) << std::endl;
```
So if fact the macro is useless code
if someone wishes name of the executable
`std::cout << argv[0] << std::endl;` does exactly the same
or if someone wishes name of function
`std::cout << "<" __func__ << ">" << std::endl;`.